buildenv/jenkins/jobs/infrastructure/artifactoryCheck.groovy

```diff
@@ -0,0 +1,73 @@
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+def LABEL = (params.LABEL) ? params.LABEL : "artifactory"
+def WARNING_THRESHOLD = (params.WARNING_THRESHOLD) ? params.WARNING.toInteger() : 75
+def DANGER_THRESHOLD = (params.DANGER_THRESHOLD) ? params.DANGER_THRESHOLD.toInteger() : 90
+def SLACK_CHANNEL = (params.SLACK_CHANNEL) ?: "#jenkins-sandbox"
+def SLACK_MESSAGE = "${JOB_NAME} jenkins job\n"
+def HIGH_ROOT_DISK_PERCENTAGE = 0
+def HIGH_ARTIFACTORY_DISK_PERCENTAGE = 0
+def HIGH_NODE_ROOT
+def HIGH_NODE_ARTIFACTORY
+
+timestamps {
+	for (sNode in jenkins.model.Jenkins.instance.getLabel(LABEL).getNodes()) {
+		node(sNode.getNodeName()) {
+			stage("Artifactory data collection") {
+				// since there are only artifactory nodes in 2 geos, its either one or the other
+				def artifactoryGeo = (sNode.getLabelString().contains('ci.geo.unb')) ? "UNB" : 'OSU'
+				def rootDiskDetails = sh (label: 'Get / disk details', returnStdout: true, script: "df / -h | tail -1 | awk '{print \$3 \" / \" \$2}'").trim()
+				def rootDiskPercentage = sh (label: 'Get / disk percentage', returnStdout: true, script: "df / -h | tail -1 | awk '{print \$5}'").trim()
+				def artifactoryDiskDetails = sh (label: 'Get Artifactory disk details', returnStdout: true, script: "df -h | grep jenkins | awk '{print \$3 \" / \" \$2}'").trim()
+				def artifactoryDiskPercentage = sh (label: 'Get Artifactory disk percentage', returnStdout: true, script: "df -h | grep jenkins | awk '{print \$5}'").trim()
+				def message = "\nArtifactory on ${sNode.getNodeName()} hosted at ${artifactoryGeo}:\n  Root disk used: ${rootDiskDetails} -> ${rootDiskPercentage}\n  Data disk used: ${artifactoryDiskDetails} -> ${artifactoryDiskPercentage}\n"
+				SLACK_MESSAGE += "${message}"
+				echo "${message}"
+				if (HIGH_ROOT_DISK_PERCENTAGE < (rootDiskPercentage - '%').toInteger()) {
+					HIGH_ROOT_DISK_PERCENTAGE = (rootDiskPercentage - '%').toInteger()
+					HIGH_NODE_ROOT = sNode.getNodeName()
+				}
+				if (HIGH_ARTIFACTORY_DISK_PERCENTAGE < (artifactoryDiskPercentage - '%').toInteger()) {
+					HIGH_ARTIFACTORY_DISK_PERCENTAGE = (artifactoryDiskPercentage - '%').toInteger()
+					HIGH_NODE_ARTIFACTORY = sNode.getNodeName()
+				}
+			}
+		}
+	}
+
+	stage("Artifactory data review") {
+		def slackStatus = "Good"
+		
+		if ((HIGH_ROOT_DISK_PERCENTAGE > DANGER_THRESHOLD) || (HIGH_ROOT_DISK_PERCENTAGE > DANGER_THRESHOLD)) {
+			slackStatus = "Danger"
+		}
+		if ((HIGH_ARTIFACTORY_DISK_PERCENTAGE > WARNING_THRESHOLD) || (HIGH_ARTIFACTORY_DISK_PERCENTAGE > WARNING_THRESHOLD)) {
+			slackStatus = "Warning"
+		}
+		message = "\nHighest root disk percentage ${HIGH_ROOT_DISK_PERCENTAGE}% on ${HIGH_NODE_ROOT}\nHighest artifactory disk percentage ${HIGH_ARTIFACTORY_DISK_PERCENTAGE}% on ${HIGH_NODE_ARTIFACTORY}\n"
```

Comment:
`\n` at the beginning of the `message` not needed.

Reply:
I will change with the next push.